test/scheduler/utils_test.go

```diff
@@ -89,6 +89,9 @@ var (
 	lessFuncClusterDecision = func(decision1, decision2 placementv1beta1.ClusterDecision) bool {
 		return decision1.ClusterName < decision2.ClusterName
 	}
+	ignoreUnSelectedClusterDecision = func(decision placementv1beta1.ClusterDecision) bool {
+		return decision.Selected
```

Comment:
```suggestion
		return !decision.Selected
```
https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#IgnoreSliceElements
" Elements are ignored if the function reports true."

you're ignoring the selected clusters?